src/applications/check-in/travel-claim/pages/travel-agreement/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router';
+
+import Wrapper from '../../../components/layout/Wrapper';
+import BackButton from '../../../components/BackButton';
+import { useFormRouting } from '../../../hooks/useFormRouting';
+import { URLS } from '../../../utils/navigation';
+
+const TravelAgreement = props => {
+  const { router } = props;
+  const { t } = useTranslation();
+
+  const { goToPreviousPage } = useFormRouting(router);
+
+  return (
+    <>
+      <BackButton
+        router={router}
+        action={goToPreviousPage}
+        prevUrl="#back"
+        text={t('back-to-last-screen')}
+      />
+      <Wrapper
+        pageTitle="Travel Agreement"
```

Comment:
Hard coded text here and its the wrong title. Should be: "Beneficiary travel agreement"